packages/driver/cypress/integration/e2e/multidomain_spec.ts

```diff
@@ -43,11 +40,114 @@ describe('multidomain', { experimentalSessionSupport: true }, () => {
     })
   })
 
+  describe('data argument', () => {
+    it('passes object to callback function', () => {
+      cy.switchToDomain('foobar.com', { foo: 'foo', bar: 'bar' }, ({ foo, bar }) => {
+        expect(foo).to.equal('foo')
+        expect(bar).to.equal('bar')
+      })
+    })
+
+    it('passes array to callback function', () => {
+      cy.switchToDomain('foobar.com', ['foo', 'bar'], ([foo, bar]) => {
+        expect(foo).to.equal('foo')
+        expect(bar).to.equal('bar')
+      })
+    })
+
+    it('passes string to callback function', () => {
+      cy.switchToDomain('foobar.com', 'foo', (foo) => {
+        expect(foo).to.equal('foo')
+      })
+    })
+
+    it('passes number to callback function', () => {
+      cy.switchToDomain('foobar.com', 1, (num) => {
+        expect(num).to.equal(1)
+      })
+    })
+
+    it('passes boolean to callback function', () => {
+      cy.switchToDomain('foobar.com', true, (bool) => {
+        expect(bool).to.be.true
+      })
+    })
+
+    describe('errors', () => {
+      it('errors if experimental flag is not enabled', (done) => {
+        cy.on('fail', (err) => {
+          expect(err.message).to.equal('`cy.switchToDomain()` requires enabling the experimentalMultiDomain flag')
+
+          // @ts-ignore
+          Cypress.config('experimentalMultiDomain', true)
+
+          done()
+        })
+
+        // @ts-ignore
+        Cypress.config('experimentalMultiDomain', false)
+
+        // @ts-ignore
+        cy.switchToDomain()
+      })
```

Comment:
Suggest setting the experimental flag in the `it`
```suggestion
      // @ts-ignore
      it('errors if experimental flag is not enabled', { experimentalMultiDomain: false }, (done) => {
        cy.on('fail', (err) => {
          expect(err.message).to.equal('`cy.switchToDomain()` requires enabling the experimentalMultiDomain flag')

          done()
        })

        // @ts-ignore
        cy.switchToDomain()
      })
```

Reply:
Good call 💯